docs/root/version_history/current.rst

```diff
@@ -102,6 +102,7 @@ Bug Fixes
 * tls: fix issue where OCSP was inadvertently removed from SSL response in multi-context scenarios.
 * upstream: fix handling of moving endpoints between priorities when active health checks are enabled. Previously moving to a higher numbered priority was a NOOP, and moving to a lower numbered priority caused an abort.
 * upstream: retry budgets will now set default values for xDS configurations.
+* xray: fix the default sampling 'rate' for AWS X-Ray tracer extension to be 5%.
```

Comment:
Is there a reason this is under "bug fixes" and not "minor behavioral changes"?

Reply:
As per the design proposal [#5644](https://github.com/envoyproxy/envoy/issues/5644) and documented behavior, the default sampling rate for AWS X-Ray tracer is expected to be 5%. It was a typo/bug in the code that the _DefaultRate_ value was 0.5 (50%) and not 0.05 (5%). Which will be addressed by this PR.

Reply:
Consider appending something like "as opposed to 50%" so it's clear that was the bug.